src/include/baselib/jni/JavaVirtualMachineConfig.h

```diff
@@ -49,6 +50,28 @@ namespace bl
             {
                 m_libraryPath = BL_PARAM_FWD( libraryPath );
             }
+
+            const std::string& getClassPath() const NOEXCEPT
+            {
+                return m_classPath;
+            }
+
+            void setClassPath( SAA_in std::string&& classPath )
+            {
+                m_classPath = BL_PARAM_FWD( classPath );
+            }
+
+            std::vector< std::string > getJavaVMOptions()
+            {
+                std::vector< std::string > options;
+
+                if( ! m_classPath.empty() )
+                {
+                    options.push_back( "-Djava.class.path=" + m_classPath );
```

Comment:
Checked with and without quotes. The JVM will actually treat the quotes around the path after the '=' sign as if they are part of the path itself.